qa/automation/CareerPortal.Tests.Scenarios/Burlaka.feature

```diff
@@ -0,0 +1,119 @@
+And user should be able to download a "Quantori Privacy Policy" pdf file
+
+Scenario:1.2.3_04 Footer link "Cookies Settings" opens a pop-up window with cookie settings
+Given user opened "career.quantori.com"  
+And user scrolled to the bottom of the "career.quantori.com" page
+When user clicks a link with "Cookies Settings" text in the footer
+Then a pop-up window with the following text should be displayed in the same window:
+"""
+We use cookies to enhance the functionality of the website. 
+Some cookies are necessary for our site to work while others are optional to aid in the user experience
+For overall information about how we protect your data please review our Data Privacy Policy.
+"""
+
+Scenario:1.2.3_05 Footer link "Cookies Settings" allows a user to submit selected by default (necessary) cookies preferences
+Given user opened "career.quantori.com"  
+And user scrolled to the bottom of the "career.quantori.com" page
+And user clicked a link with "Cookies Settings" text in the footer
+When a pop-up window with selected by default "Necessary cookies" checkbox and "Other cookies" checkbox is displayed
+Then user should click "Submit preferences" button
+
+Scenario:1.2.3_06 Footer link "Cookies Settings" allows a user to choose and submit other cookies preferences
+Given user opened "career.quantori.com"  
+And user scrolled to the bottom of the "career.quantori.com" page
+And user clicked a link with "Cookies Settings" text in the footer
+When a pop-up window with selected by default "Necessary cookies" checkbox and "Other cookies" checkbox is displayed
+Then user should select "Other cookies" checkbox 
+And user should click "Submit preferences" button
+
+Scenario:1.2.3_07 Pop-up window "Cookies Settings" contains a link to "Data Privacy policy" pdf file
+Given user opened "career.quantori.com"  
+And user scrolled to the bottom of the "career.quantori.com" page
+And user clicked a link with "Cookies Settings" text in the footer
+When a pop-up window with cookies settings is displayed
+Then a link with "Data Privacy policy" text should be displayed
+
+Scenario Outline:1.2.3_08 Footer link buttons redirect a user to the corresponding websites
+Given user opened "career.quantori.com" 
+And user scrolled to the bottom of the page
+When user clicks a link button with a "<logo>" logo in the footer
+Then user should be redirected to "Quantori page" on the "<full website name>" website
+
+Examples:
+|logo                   |full website name     | 
+|"Planet icon"          |Quantori.com          | 
+|hh                     |hh.ru                 | 
+|f                      |facebook.com          | 
+|"Camera icon"          |instagram.com         |
+|vk                     |vk.com                |
+|in                     |linkedin.com          |
+
+Scenario:1.2.3_09 Footer link button does not redirect a user with Ukrainian IP address to Quantori page on vk.com website
+Given user did not change his Ukrainian IP address with VPN services 
```

Comment:
Thank you for the comment!)
I guess the most likely situation is that a user has a Ukrainian IP by default (if he accesses the Quantori website from Ukraine) rather than he sets his local machine to the Ukrainian IP address.

Reply:
I thought that scenario should be written in general for person who can live in any places)))

Reply:
I guess this is only an example of some negative scenario :).